docs/development/local-development.md

```diff
@@ -46,7 +46,7 @@ If pushing to an external image registry, you may need to provide credentials to
 
 ```bash
 $ make ko
-$ ko login <IMAGE_REGISTRY> -u <USERNAME> -p <PASSWORD>
+$ ko auth login <IMAGE_REGISTRY> -u <USERNAME> -p <PASSWORD>
```

Comment:
Based on [their code around](https://github.com/google/ko/blob/516cdeec7d37f8ac4697411beb8f04ec3968184a/main.go#L44-L49) that, I think we should stick with `login`.

```suggestion
$ ko login <IMAGE_REGISTRY> -u <USERNAME> -p <PASSWORD>
```

Reply:
Here I'm on commit `ea5dc20`, and it returns:

```
$ ko login
Error: unknown command "login" for "ko"
Run 'ko --help' for usage.
2021/05/06 11:46:02 error during command execution: unknown command "login" for "ko"
```

Given more users can be sharing the same version, should we consider using the new style `ko auth ...` instead?

Reply:
I am confused now. Which version of `ko` are you using? Maybe my confusion is that I am on the latest version and there the `login` command is available and `auth` is hidden. With that being said, to be backwards compatible, you are right, we should use `auth` and maybe later down the road can eventually switch to `login`.

Reply:
I disagree - our make target installs `ko` at a more recent version, the removal of `auth` means that the command may very well disappear in a future release. Ko is not at v1 yet, so it is safe to assume developers can (and should) upgrade their tooling.

Reply:
Well, given the common way is `ko login`, I'm reverting this change. I think my local `ko` binary was built via `go get` before `master` branch was renamed to `main` and thus it got stale. Thanks for the inputs, folks!